docs/model-types/README.md

```diff
@@ -5,18 +5,22 @@ By leveraging existing third-party model servers, we support a number of standar
 - [Keras](keras.md)
 - [LightGBM](lightgbm.md)
 - [ONNX](onnx.md)
+- [OpenVINO IR](openvino-ir.md)
 - [PyTorch ScriptModule](pytorch.md)
 - [scikit-learn](sklearn.md)
 - [TensorFlow](tensorflow.md)
 - [XGBoost](xgboost.md)
 
-| Model Type | Framework    | Supported via ServingRuntime |
-| ---------- | ------------ | ---------------------------- |
-| keras      | TensorFlow   | Triton (C++)                 |
-| lightgbm   | LightGBM     | MLServer (python)            |
-| onnx       | ONNX         | Triton (C++)                 |
-| pytorch    | PyTorch      | Triton (C++)                 |
-| sklearn    | scikit-learn | MLServer (python)            |
-| tensorflow | TensorFlow   | Triton (C++)                 |
-| xgboost    | XGBoost      | MLServer (python)            |
-| \*         | Custom       | [Custom](../runtimes) (any)  |
+| Model Type  | Framework        | Supported via ServingRuntime |
+| ----------- | ---------------- | ---------------------------- |
+| keras       | TensorFlow       | Triton (C++)                 |
+| lightgbm    | LightGBM         | MLServer (python)            |
+| onnx        | ONNX             | Triton (C++), OVMS (C++)     |
+| openvino_ir | Intel OpenVINO\* | OVMS (C++)                   |
+| pytorch     | PyTorch          | Triton (C++)                 |
+| sklearn     | scikit-learn     | MLServer (python)            |
+| tensorflow  | TensorFlow       | Triton (C++)                 |
+| xgboost     | XGBoost          | MLServer (python)            |
+| \*\*        | Custom           | [Custom](../runtimes) (any)  |
+
```

Comment:
Looks good, thanks!
I thought the footnote is mainly for the new built-in OVMS runtime. Not sure the use of double asterisk symbols for the row that is for custom runtimes. In fact, models in many ml frameworks can be converted to ONNX IR and run in Triton or OVMS. I would recommend remove the second asterisk.

Reply:
I think the asterisk in custom row originally indicated "any", so I agree it doesn't make sense to also use an asterisk for a footnote indicator in the same table. Maybe use a superscript `1` if that's possible in markdown, or otherwise some other symbol.

Reply:
Looks good. Thanks!